test/test.js

```diff
@@ -39,15 +39,40 @@ Object.keys(cases).forEach((name) => {
 
     let resultJson;
 
+    const rootsSeenBeforePlugin = new Set();
+    const rootsSeenAfterPlugin = new Set();
+
     const plugins = [
       autoprefixer,
+      postcss.plugin(
```

Comment:
Could you extract this test in a separate test case to not mess up the other tests?

Reply:
can do but I thought this was a good thing to run assert for all the `cases`?

Reply:
I made it a new test for all the cases. This ok?

Reply:
Yeah, totally ok